pom.xml

```diff
@@ -12,7 +12,7 @@
     <artifactId>uv-l-rdfToVirtuosoAndCkan</artifactId>
     <name>L-RdfToVirtuosoAndCkan</name>
     <description>Loads RDF data to Virtuoso using L-RdfToVirtuoso and creates CKAN resources using L-RdfToCkan.</description>
-    <version>1.1.6-SNAPSHOT</version>
+    <version>1.2.0-SNAPSHOT</version>
```

Comment:
@eea03 this version shall be also in debian/pom.xml in dependencies part for this DPU